interface/MuonsProducer.h

```diff
@@ -41,6 +42,7 @@ class MuonsProducer: public LeptonsProducer<pat::Muon>, public ScaleFactors {
 
         BRANCH(dxy, std::vector<float>);
         BRANCH(dz, std::vector<float>);
+        rochcor2015 rmcor;
```

Comment:
It seems to break seg if it is set to private (which was the first thing I tried)